.travis.yml

```diff
@@ -18,7 +18,6 @@ go:
 
 services:
   - docker
-  - redis-server
```

Comment:
?

Reply:
Yep! Redis started by the both docker and local exec runners, so the tests will pass without redis-server running on the host.

Reply:
The local:exec runner starts the Redis server if it's available on the PATH of the host. You mean to say that the Redis server is always on the path, and all that the `redis-server` entry does here is start it?

Reply:
Closing the loop here. The answer is that the local:exec runner now runs prerequisites (redis, pushgateway, prometheus, etc.) as docker containers.